ard_controller/ard_controller.ino

```diff
@@ -0,0 +1,29 @@
+#include <DueTimer.h>
+
+bool write_flag = false;
+
+void setup() {
+  // put your setup code here, to run once:
+  Serial.begin(115200);
+  Serial1.begin(115200);
+
+  Timer1.attachInterrupt(serial_ISR).setFrequency(1).start();
+}
+
+void loop() {
+  int data;
+  if (Serial1.available() > 0) {
```

Comment:
This ping/pong behavior only happens when any data is sent and received--aka no parsing is being done?

Reply:
Correct. It doesn't look at the data, just checks for data being transmitted/received

Reply:
ack